scvi/module/_vae.py

```diff
@@ -518,16 +553,20 @@ def sample(
 
     @torch.inference_mode()
     @auto_move_data
-    def marginal_ll(self, tensors, n_mc_samples):
+    def marginal_ll(
+        self, tensors, n_mc_samples, observation_specific=False, n_mc_samples_per_pass=1
```

Comment:
n_mc_samples_per_pass is like chunking, we need to validate that the chunking is less than n_mc_samples?